src/pages/index.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+    <Choropleth />
+    <div class="sticky-footer">
+        <div v-for="name in indicatorNames" :key="name">
+            <v-btn
+                link
+                class="mb-2"
+                :class="name === selectedIndicator ? 'bg-blue' : 'bg-black'"
```

Comment:
also grey text? otherwise, especially with two indicators it's not totally obvious which is active